requirements/edx-sandbox/py35.txt

```diff
@@ -62,7 +62,6 @@ numpy==1.16.5
     #   chem
     #   matplotlib
     #   openedx-calc
-    #   scipy
```

Comment:
Minor comment: Seems weird that this line was changed? Doesn't matter though, as I see it's a comment.